extensions/aws/s3/provision/src/main/java/org/eclipse/dataspaceconnector/provision/aws/s3/S3BucketProvisionerConfiguration.java

```diff
@@ -0,0 +1,55 @@
+package org.eclipse.dataspaceconnector.provision.aws.s3;
```

Comment:
Why does this class need a builder, it only has two parameters?

Also, I don't think the service extension context should be passed to it. That can be handled in the extension initialize (I would also trap the Integer parse and report the problem if an exception is thrown)

Reply:
Done. 
Regarding the Builder, there's not a clear rule about the minimum parameter size, but I agree that this was pretty overkill.